meeting-notes/2024/2024-feb-01-meeting-notes.md

```diff
@@ -0,0 +1,41 @@
+# 2024 February 2 - Meeting notes
+
+## Meeting recording
+
+Watch the YouTube recording of the meeting: https://www.youtube.com/watch?v=1ZKq6ohi8Ko&list=PL9svBjLDUl_-gyXj2lDYomI8keXtbMqUl
+
+## Agenda and notes
+
+* **Introductions** - Tell us about yourself, where you're at in the world, how you use Salt and your experience level, why you are interested in this group, and what you hope to get out of it.
+- **How we work**
+  - Decision: We will record the meeting and let it be posted on YouTube.
+  - Decision: We will keep meeting notes in MD in a community repository.
+  - Action: Nick will create the community repository and Alyssa will post the meeting notes.
+- **Publishing packages to PyPi**
+  - We had a discussion about publishing packages to PyPi. Jaimie wanted to ensure there is redundancy among the PyPi maintainers to ensure consistent access is maintained.
+  - Everything is published by `salt-extensions-community` in PyPi. Nick has the credentials in his company account and can add others if needed.

```

Comment:
**issue (blocking):** fix PyPi user name

```suggestion
  - Everything is published by [saltext-community](https://pypi.org/user/saltext-community/) in PyPi. Nick has the credentials in his company account and can add others if needed.

```

Reply:
I've adopted this suggestion.